README.md

```diff
@@ -1,9 +1,32 @@
+<div align="center">
+<img width="50%" align="center" style="display: block; "
+     src="https://blog.ariga.io/uploads/images/atlas_logo_text.png"/>
+</div>
+</br>
+<h1 align="center">A Database Toolkit</h1>
 
-# Atlas -  A Database Toolkit
-[![Discord](https://img.shields.io/discord/930720389120794674?label=discord&logo=discord&style=flat-square&logoColor=white)](https://discord.gg/zZ6sWVg6NT)
 
-<img width="50%" align="right" style="display: block; margin:40px auto;"
-     src="https://atlasgo.io/uploads/images/gopher.png"/>
+<div align="center">
+Manage your database schemas with Atlas CLI
+</div>
+
+<div align="center">
+     <p align="center">
+         <br />
+         <a href="https://atlasgo.io/getting-started/?utm_source=github&utm_campaign=github_atlas_readme&utm_medium=social" rel="dofollow"><strong>Explore the docs »</strong></a>
+         <br /><br/>
+          <a href="https://discord.com/invite/zZ6sWVg6NT">Discord</a>
+    ·
+          <a href="https://twitter.com/ariga_io/?utm_source=github&utm_campaign=github_atlas_readme&utm_medium=social">Twitter</a>
+     ·
+          <a href="https://github.com/ariga/atlas/issues">Report a Bug</a>
+    ·
+          <a href="https://github.com/ariga/atlas/discussions">Request a Feature</a>
```

Comment:
Both links should point to a creation of a new issue, but with option for selecting a template (from issue template). See ent as an example please: https://github.com/ent/ent/tree/master/.github/ISSUE_TEMPLATE

Reply:
Do you want this in this PR or is it a different one?

Reply:
Whatever you prefer.